modules/apps/commerce/commerce-payment-service/src/main/resources/META-INF/portlet-model-hints.xml

```diff
@@ -12,10 +12,20 @@
 		<field name="classNameId" type="long" />
 		<field name="classPK" type="long" />
 		<field name="amount" type="BigDecimal" />
+		<field name="callbackURL" type="String">
+			<hint-collection name="CLOB" />
+		</field>
+		<field name="channelId" type="long" />
```

Comment:
Have to sort channelId up to match service.xml since we are changing the sorting there.